### DISCUSSION TITLE
多个角色UP池分开t统计

### DISCUSSION THREAD

**Discussion by linyu003** [sunfkny/genshin-gacha-export#52] [功能建议 / Ideas]
2.7版本有两个角色UP池，希望能分开统计，而不是合并在一起。

**Comment by sunfkny** [comment]
目前所有的抽卡统计工具都是合在一起的，因为共享保底的卡池分开来不便于统计，如果你确实有需求，可以使用原始数据 sheet 或者UIGF json中的gacha_type来区分

gacha_type 对应的卡池名称：
100 新手祈愿
200 常驻祈愿
301 角色活动祈愿
400 角色活动祈愿-2
302 武器活动祈愿
